go.mod

```
@@ -5,7 +5,7 @@ go 1.20
 require (
 	github.com/google/go-cmp v0.6.0
 	github.com/goplus/gogen v1.16.4
-	github.com/goplus/llgo v0.10.0
+	github.com/goplus/llgo v0.10.1
```

```suggestion
	github.com/goplus/llgo v0.10.0
```
can resolve this fail https://github.com/goplus/llcppg/actions/runs/14370680609/job/40294196011?pr=232